bin/crew

```diff
@@ -1328,6 +1328,14 @@ def install
     puts "Package #{@pkg.name} already installed, skipping...".lightgreen
     return
   end
+    
+  if @pkg.deprecated?
+    puts "Package #{@pkg.name} is deprecated".orange
+    if @pkg.in_upgrade
+       puts "\b, skipping...".orange
```

Comment:
Why the backspace?

Reply:
to remove the new line

Reply:
You can use `print` instead of `puts` to avoid the new line at the end.